### DISCUSSION TITLE
Cookie Banner

### DISCUSSION THREAD

**Discussion by AliZaibazr** [maize-tech/laravel-legal-consent#2] [Q&A] [answered]
Hi maize tech, I see it seems very helpful.
My concern is about cookie banner, does it also have feature of cookie banner or will be in future realse?

**Comment by riccardodallavia** [comment, marked as answer]
Hi Ali!

You can enable the package for cookie banners too!
All you have to do is add the new document type in the configs like so:

```php
'allowed_document_types' => [
    'privacy-policy',
    'terms-of-use',
    'cookie-policy', // here is the new document type
],
```

and create the cookie policy text as explained in the [readme](https://github.com/maize-tech/laravel-legal-consent#basic)!

Let me know if that's what you were expecting 😉
